front/src/modules/SimulationReportSheet/Route.tsx

```diff
@@ -0,0 +1,124 @@
+  const { mode } = props;
+  const isStdcm = mode === 'stdcm';
+
+  const rows = formatRouteTable({ ...props });
+
+  return (
+    <View style={styles.consistAndRoute.stopTableContainer}>
+      <Table style={styles.consistAndRoute.stopTable}>
+        <TH style={styles.consistAndRoute.stopTableTH}>
+          <View style={styles.consistAndRoute.stopTableIndexWidth}>
+            <TD aria-label="line-count" />
+          </View>
+          <View style={styles.consistAndRoute.stopTableOpWidth}>
+            <TD>{t('reportSheet.operationalPoint')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableChWidth}>
+            <TD>{t('reportSheet.code')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableEndWidth}>
+            <TD>{t('reportSheet.endStop')}</TD>
+          </View>
+          {isStdcm && (
+            <View style={styles.consistAndRoute.stopForWidth}>
+              <TD>{t('reportSheet.stopTime')}</TD>
+            </View>
+          )}
+          <View style={styles.consistAndRoute.stopTableStartWidth}>
+            <TD>{t('reportSheet.startStop')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableStopTypeWidth}>
+            {isStdcm && <TD>{t('reportSheet.stopType')}</TD>}
+          </View>
+        </TH>
+
+        {rows.map((row, idx) => {
+          const isLast = idx === rows.length - 1;
+          const isFirst = idx === 0;
+          return (
+            <TR key={row.index} style={styles.consistAndRoute.stopTableTbody}>
+              <View style={styles.consistAndRoute.stopTableIndexWidth}>
+                <TD style={styles.consistAndRoute.stopTableIndexColumn}>{row.index}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableOpWidth}>
+                <TD style={styles.consistAndRoute.stopTableOpColumn}>{row.name}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableChWidth}>
+                <TD style={styles.consistAndRoute.stopTableChColumn}>{row.secondaryCode}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableEndWidth}>
+                <TD
+                  style={
+                    row.italic
+                      ? styles.consistAndRoute.stopTableItalicColumn
+                      : styles.consistAndRoute.stopTableStartColumn
+                  }
+                >
+                  {row.arrivesAt}
+                  {row.tolerances && isLast && (
+                    <View style={styles.consistAndRoute.tolerancesWidth}>
+                      {row.tolerances.map((tolerances, i) => (
+                        <Text key={i} style={styles.consistAndRoute.tolerancesText}>
+                          {tolerances}
+                        </Text>
+                      ))}
+                    </View>
+                  )}
+                </TD>
+              </View>
+              {isStdcm && (
```

Comment:
You can check here directly if the row has a passageStop (by design, you won't have a passageStop if you're not in stdcm mode, see your function formatRouteTable)
```suggestion
              {row.passageStop && (
```

Reply:
I'd like to, but the columns are all out of line if I do that: 
<img width="725" alt="Capture d’écran 2025-06-23 à 11 22 07" src="https://github.com/user-attachments/assets/f6c965b7-bea1-4047-a724-fd6e7cbce700" />